themes/_core/js/product.js

```diff
@@ -154,6 +154,7 @@ function updateProduct(event, eventType, updateUrl) {
       url: updateUrl + ((updateUrl.indexOf('?') === -1) ? '?' : '&') + formSerialized + preview,
       method: 'POST',
       data: {
+        quickview: $('.modal.quickview.in').length,
```

Comment:
Has @matks said earlier, is there a quick way to put the `is_quick_view` smarty variable into the template to retrieve the data instead of hoping the theme is using the same modal? :)

Reply:
Probably, but as it is regression and because of related PR changes my module doesn't work (and probably some other modules), I thought that it would be better to do it without it, because this would require theme authors to make proper changed too, and it's way harder to update theme than a module 🤷🏼‍♂️

Reply:
Oh, reviewing too much pull request didn't notice the 1.7.7.x branch :sweat_smile: 
Let's improve this for the develop branch :)